docs/conventions/language_conventions.md

```diff
@@ -0,0 +1,7 @@
+## Language Conventions
+
+The entire code-base, branches, commits and issues should all be written in english.
+
+Pull Requests should be written in norwegian.
```

Comment:
Kanskje `allows both Norwegian and English`

Reply:
Enig, fixed